package.json

```diff
@@ -13,68 +13,55 @@
     "integration:local": "npm run serve",
     "integration:metamask": "npm run integration:wait && npm run integration:transfers",
     "integration:wait": "wait-on -l -i 1000 tcp:8545 http-get://localhost:5000",
-    "integration:transfers": "node ./integration/index.mjs",
+    "integration:transfers": "node --experimental-modules ./integration/index.mjs",
     "integration:dappeteer": "concurrently -k -s first \"npm:integration:blockchain\" \"npm:integration:local\" \"npm:integration:metamask\"",
-    "lint": "eslint ./src --ext .js,.ts,.tsx,.mdx --max-warnings 0",
+    "lint": "eslint ./src --ext .js,.ts,.tsx --max-warnings 0 && eslint ./integration --ext .mjs --max-warnings 0",
```

Comment:
lint:fix wont work because fix will be passed to the second command only

Reply:
issue fixed at #22